.github/workflows/build-book.yml

```diff
@@ -47,6 +47,9 @@ jobs:
           cd book
           R -q -e 'options(Ncpus = 2); install.packages("renv", repos = "cloud.r-project.org"); renv::restore()'
 
+      - name: Document and install book
+        run: make bookinstall
```

Comment:
@be-marc please note this change I've made to the workflow, it broke before because the book wasn't being documented and installed so new R functions were'nt registered. I don't *think* this breaks the changes you made but also I don't know how `renv` works so please fix if needed!